File: exercise1/src/main/java/com/exercise1/exercise1/entity/Post.java
package com.exercise1.exercise1.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


@Entity
@NoArgsConstructor
@Data
@AllArgsConstructor
public class Post {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    long id;
    String title;
    String content;
    String author;

    public Post(String title, String content, String author) {
        this.title = title;
        this.content = content;
        this.author = author;
    }
}

File: REACT/my-app/src/components/Post.js
import React from 'react';
import "./css/post.css";

const Post = (props) => {
    return (
        <div className="Content" onClick = {props.onClick}>
            <p>ID: {props.id}</p>
            <p>Title: {props.title}</p>
            <p>Author: {props.author}</p>
        </div>
    );
}

export default Post;
